Fix remote exam docs to clarify auto-close behavior (#2250)
PrairieLearn/PrairieLearn: doc/remoteExams.md

* Remember to extend both `endDate` *and* `timeLimitMin` for students with extra-time accommodations.
* Students who are scheduled for a conflict exam will be able to access the exam during the primary time slot. However, if they do so, they will be blocked from the exam during the conflict timeslot.
* After the timer expires the exam will auto-close and grade any saved but ungraded questions and show students their final score. After this time students will be unable to see any of the questions.

> * If a student closes their web browser before the exam is complete, their exam will be automatically closed and graded within 12 minutes of when their timer expires. If they try and access their exam during this time it will immediately close and grade.

Reviewer: is this within 12 minutes *after* the timer expires?

Author: Yep, how can we word that more clearly?

Reviewer: "within 12 minutes after their timer expires" :stuck_out_tongue:

> * If a student closes their web browser before the exam is complete, their exam will be automatically closed and graded within 12 minutes of when their timer expires. If they try and access their exam during this time it will immediately close and grade.

Author: ```suggestion
* If a student closes their web browser before the exam is complete, their exam will be automatically closed and graded within 12 minutes after their timer expires. If they try and access their exam during this time it will immediately close and grade.
```
